dev-requirements.txt

```diff
@@ -1,4 +1,5 @@
 flake8
```

Comment:
nit: include `-r requirements.txt` at the top?

(that way you only have to `pip include -r ` one file.